DeepCat.js

```diff
@@ -139,14 +139,14 @@
 	}
 
 	function getAjaxRequest( searchTerm, searchTermNum ) {
-		var categoryString = DeepCat.extractDeepCatCategory( searchTerm ),
-			userParameter = {
+		({  categoryString, depth } = DeepCat.extractDeepCatCategory( searchTerm ));
```

Comment:
I'm not that into js so I wasn't and am still not sure whats the most elegant way to handle that.